### PR TITLE
Add security scan information to the release list response

### PR DESCRIPTION
**Pull request by lpuskas**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    |yes
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?
This PR implements decoration of release list with security scan information

### Why?
This information was provided in the earlier versions as well

### CODE REVIEW

File: internal/helm/service.go [hunk 1]
```diff
@@ -537,7 +540,7 @@ func (s service) UpgradeRelease(ctx context.Context, organizationID uint, cluste
 	return nil
 }
 
-func (s service) ListCharts(ctx context.Context, organizationID uint, filter ChartFilter, options Options) (charts ChartList, err error) {
+func (s service) ListCharts(ctx context.Context, organizationID uint, filter ChartFilter, _ Options) (charts ChartList, err error) {
```
Comment (pepov) [inline internal/helm/service.go:540]: Not sure where these changes come from but doesn't seem like it's necessary

File: internal/helm/service.go [hunk 2]
```diff
@@ -551,7 +554,7 @@ func (s service) ListCharts(ctx context.Context, organizationID uint, filter Cha
 	return chartList, nil
 }
 
-func (s service) GetChart(ctx context.Context, organizationID uint, chartFilter ChartFilter, options Options) (chartDetails ChartDetails, err error) {
+func (s service) GetChart(ctx context.Context, organizationID uint, chartFilter ChartFilter, _options Options) (chartDetails ChartDetails, err error) {
```
Comment (pepov) [inline internal/helm/service.go:554]: Not sure where these changes come from but doesn't seem like it's necessary

File: internal/helm/service.go (hunk 1 above)
Comment (lpuskas) [reply internal/helm/service.go:537]: this is linter a linter fix

File: internal/helm/service.go (hunk 2 above)
Comment (lpuskas) [reply internal/helm/service.go:551]: linter fix

**Pull request merged**